src/backend/tests/unmocked/recruitment.test.ts

```
@@ -88,4 +89,41 @@ describe('Recruitment Tests', () => {
       expect(result).toStrictEqual([milestone1, milestone2]);
     });
   });
+
+  describe('Delete a single milestone', () => {
+    it('Fails if user is not admin', async () => {
+      await expect(
+        async () => await RecruitmentServices.deleteMilestone(await createTestUser(wonderwomanGuest, orgId), 'id', orgId)
+      ).rejects.toThrow(new AccessDeniedAdminOnlyException('delete milestone'));
+    });
+
+    it('Fails if milestoneId is not found', async () => {
+      await expect(
+        async () => await RecruitmentServices.deleteMilestone(await createTestUser(batmanAppAdmin, orgId), 'id1', orgId)
+      ).rejects.toThrow(new HttpException(400, 'Milestone with id: id1 not found!'));
```

thijs should throw a not found exception